README.md

```diff
@@ -19,9 +19,9 @@ Read our [Download and Installation](https://www.xwiki.org/xwiki/bin/view/Downlo
 Read our [Release Notes](https://www.xwiki.org/xwiki/bin/view/ReleaseNotes/).
 
 ## Tools
-* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit
-* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI) if you want to report an issue
-* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build XWiki
+* [Continuous Integration](https://ci.xwiki.org/) setup launches a build for each commit.   
+* [Issue Tracker](https://jira.xwiki.org/browse/XWIKI), if you want to report an issue.
+* [Development Flow](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HGeneralDevelopmentFlow) to see the full list of tools we use to build "XWiki".
```

Comment:
I don't think we need quotes here. That would be such a pain to do this wherever XWiki is mentioned ;)

Reply:
Ok,
But I think we don't need to do this everywhere but at that line, it looks perfect.

Reply:
What about full stop and comma ? 
I can remove quotes if you like to update 
( . Or , ) .

Reply:
Maybe we could write `to build the XWiki software` if you think that's better?

Reply:
Full stop and commas look ok.

Reply:
Hmm....it's also looks good. 
We can do this.

Reply:
Ok 🆗
That's my very first open source contribution/pr
Starting my journey with xwiki.
I'll definitely going to contribute more valuable!!